modules/test/playwright/tests/frontend-data-set-admin-web/main/helpers/DataSetManagerApiHelpers.ts

```diff
@@ -35,9 +29,9 @@ export class DataSetManagerApiHelpers extends ApiHelpers {
 		erc = 'sampleDataSetERC',
 		label = DEFAULT_LABEL.DATA_SET,
 		listOfItemsPerPage = '4, 8, 20, 40, 60',
-		restApplication = `${API_ENDPOINT_PATH}/table-sections`,
+		restApplication = API_ENDPOINT_PATH,
```

Comment:
With this change, we are no longer using table sections, rather data sets. This may be a mistake, as it has way too many consequences. Note, this is still WIP.

The reason why I started doing this is because DSM does not generate API urls correctly. We are creating them from RESTApplication / RESTSchema / RESTEndpoint, but the same simple pattern fails when there is a root model. Basically, DSM incorrectly creates fragment for visualizations of children in root model hierarchy. I was hoping this is something we can take care of independently of this PR. At the moment, I am not sure if that is wise.

Reply:
I force-pushed the solution without tests, as
1. My approach was causing way too many regressions.
2. There are talks of changing API again, that would invalidate those test changes.

Without tests, the PR can still serve for manual testing.